docs/source/tutorials/qiime2_docker.md

```diff
@@ -0,0 +1,89 @@
+# Getting Qiime 2024.5 via container
+
+## Setting up your `.bashrc` config for Apptainer
+
+[Apptainer](https://apptainer.org/) is the system on SAGA that allows you to use containers.
+
+Some configuration is necessary to make it easier to use containers
```

Comment:
Also add the sentence: "To do that we need to modify the file `.bashrc`, which is located in your home directory.

Reply:
ok